Flagsmith.FlagsmithClient/FlagsmithClient.cs

```diff
@@ -74,7 +81,9 @@ public class FlagsmithClient : IFlagsmithClient
             Dictionary<string, string> customHeaders = null,
             int retries = 1,
             double? requestTimeout = null,
-            HttpClient httpClient = null)
+            HttpClient httpClient = null,
+            bool cachingEnabled = false,
+            int cacheDurationInMinutes = 5)
```

Comment:
I would rather properties like this were contained within a separate `CacheConfig` class for example to make it more easily extensible without having to change the constructor on the main client class.